tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
Won't there be a hotfix branch on the upstream that hotfixes would get merged into? Or is that not how they traditionally work?

We'll also need to consider whether we want to have hotfix branches deployed onto staging first... or do you think hotfixes are usually enough of an emergency situation to merit going straight to production?

Can it work like this? Most recent commit to `[hotfix|master]', untagged, goes to staging. Most recent commit to '[hotfix|master]', tagged for a release, goes to production. And when I say "most recent," it's just to imply how this works over time, but in reality, it just means "a commit". So stable commits supersede hotfix commits by virtue of existing after/getting deployed after a hotfix commit. And vice versa.